pkg/arvo/app/chat-hook.hoon

```diff
@@ -402,41 +402,15 @@
   ?>  (~(has by synced) pas)
   ::  check if read is permitted
   ?>  (is-permitted src.bol pas)
+  =/  =mailbox  (need (chat-scry pas))
```

Comment:
This is a nit, but since you don't use `.mailbox`, I'd just do this:
```
=/  envs  envelopes:(need (chat-scry pas))
```

Reply:
I like this! Thanks